CMakeLists.txt

```diff
@@ -42,6 +42,13 @@ if (DEFINED ENV{JOB_ID})
   set(BUILD_SEQ $ENV{JOB_ID})
 else ()
   set(BUILD_SEQ "dev")
+  if (DEFINED ENV{PR_NUMBER})
+    set(PR_BUILD "true")
+    set(BASE_URL "http://s3.amazonaws.com/hifi-public/pr-builds/$ENV{PR_NUMBER}")
+  else ()
+    set(PR_BUILD "false")
+    set(BASE_URL "http://s3.amazonaws.com/hifi-public")
+  endif ()
 endif ()
 
```

Comment:
This has merged already, but I note that **BASE_URL** is only ever defined in the **else** block above, not when the **if** check evaluates to **true**.  Is that the intended behavior?